CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+# Contributing to Spicetify-cli
+
+## Table of Contents
+
+- [I Have a Question](#i-have-a-question)
+- [How to Contribute](#how-to-contribute)
+  - [Reporting Bugs](#reporting-bugs)
+  - [Suggesting Enhancements](#suggesting-enhancements)
+  - [Your First Code Contribution](#your-first-code-contribution)
+  - [Improving The Documentation](#improving-the-documentation)
+  - [Commit Message Format](#commit-message-format)
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Documentation](https://spicetify.app/docs/getting-started/).
+
+Before you ask a question, it is best to search for existing [issues](https://github.com/spicetify/spicetify-cli/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Open an [issue](https://github.com/spicetify/spicetify-cli/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions (go, nodejs, npm, etc), depending on what seems relevant.
```

Comment:
again, we don't need this info. Only spotify and spicetify version most of the time

Reply:
removed and replace with spotify and spicetify version